test/conftest.py

```diff
@@ -107,6 +108,36 @@
 requests_cache.install_cache(urls_expire_after={"huggingface.co": timedelta(hours=1), "*": requests_cache.DO_NOT_CACHE})
 
 
+def fail_at_version(version_major, version_minor):
+    """
+    Reminder to remove deprecated features.
+
+    ```python
+    from ..conftest import fail_at_version
+
+    @fail_at_version(1, 10)  # Will fail once Haystack version goes above 1.10
```

Comment:
The comment needs to be changed to sth like "Will fail once Haystack version is greater than or equal to 1.10". It fails already for 1.10 and not only above 1.10

Reply:
Right! Thank you :slightly_smiling_face: 

```suggestion
    @fail_at_version(1, 10)  # Will fail once Haystack version is greater than or equal to 1.10
```